### DISCUSSION TITLE
Warning in in-game logs and logs file about mods in main game Bundles folder.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#4294] [Ideas]
It turns out if you put your own mod in the game's main Bundles folder it doesn't really work. I'm not exactly sure why, but that is the case. It might just be worth having a warning about it and suggest to use the Sandbox folder. Not sure what could be used to trigger it, perhaps a quick check to see if there are any other .masterbundles or .unity3d files in that folder (and its other directories)?

**Comment by DanielWillett** [comment]
iirc the bundles folder itself isn't searched for assets (which is kinda a strange way to do it) but only the expected subfolders, so if you put it in Items or something it'll work.
With that said I'd be interested to know why only sub folders are searched

**Reply by SDGNelson** [reply]
It worked that way before the asset loading rewrite. This is actually one of the reasons the example/source files were moved to Extras - so that Bundles could be searched recursively without a special case to ignore examples.

**Comment by SDGNelson** [comment]
Can you elaborate on what you mean by not working please?

**Reply by Jdance-Media** [reply]
Assets within the Bundles folder seem to use the core masterbundle instead,resulting in the correct prefabs not being found. Like, I can have a masterbundle in a separate directory with my asset but in the Bundles folder and the asset will still use the core masterbundle.

**Reply by SDGNelson** [reply]
It *might* be because when a masterbundle is loaded, if it's the core origin it will set the global core masterbundle to the loaded one. That being said, it's only used to wait for it to load before starting further loading since other stuff depends on it.

Could you attach your example case please and I'll see what happens?
